ksqldb-engine/src/test/java/io/confluent/ksql/function/udf/string/ToBytesTest.java

```diff
@@ -40,6 +40,8 @@ public void shouldConvertHexStringToBytes() {
             is(ByteBuffer.wrap(new byte[]{33})));
         assertThat(udf.toBytes("", "hex"),
             is(ByteBuffer.wrap(new byte[]{})));
+        assertThat(udf.toBytes("1a2B", "hex"),
+            is(ByteBuffer.wrap(new byte[]{26, 43})));
```

Comment:
can you add a test with same hex but all uppercase? Just to make sure that both return the same value.